prow/scripts/deploy_and_test_kyma.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
in prow directory, our scripts use "-" and "_" as a separator, please make it consistent.